components/Filters/Filters.js

```diff
@@ -1,18 +1,25 @@
 import React from 'react';
 
-import Filter from './Filter'
+import Orientation from './Orientation';
+import SizeFilter from './SizeFilter';
+import CustomRating from './CustomRating';
+import Favourite from './Favourite';
+import Industy from './Industry'
 
 export default class Filters extends React.Component{
   constructor(props){
     super(props);
   }
 
   render(){
-    const labels = ['size', 'favourite', 'industry'];
     return(
-      <div>
-        <h1>Filters: </h1>
-        {labels.map((el) => <Filter label={el}/>)}
+      <div className='Filters'>
+        <h3>Narrow results: </h3>
+        <Orientation />
+        <SizeFilter />
```

Comment:
zauważ, że size, favourite i industry to bardzo podobne komponenty. Idealnie było by tutaj stworzyć jeden generyczny komponent